spec/views/users/index_html_erb_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'User index', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Diego', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'I´m a student', posts_counter: 0)
+    @user2 = User.create(name: 'Marcos', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                         bio: 'I´m a student', posts_counter: 1)
+    @user3 = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                         bio: 'I´m a student', posts_counter: 2)
+    visit users_path
+  end
+
+  describe 'users index page' do
+    # Display all the usernames test
+    it 'displays the created users' do
+      expect(page).to have_content('Diego')
+      expect(page).to have_content('Marcos')
+      expect(page).to have_content('Tom')
+    end
+
+    # Profile picture test
+    it 'shows the right photo' do
+      expect(page).to have_xpath("//img[@src='https://unsplash.com/photos/F_-0BxGuVvo']")
+    end
```

Comment:
- [ ] Dear @Haywayaheadshot, Good job adding the tests 🙌, But for the profile picture we need a tests that asserts you can see the profile picture for each user. since you have 3 users, Please make sure there are three pictures. 

> I suppose you use a different pictures and see if all are availabel.

Reply:
@Tadesse-Alemayehu My partner have made those corrections

Reply:
- Dear @Haywayaheadshot, Please use 3 pictures and add 3 tests to make sure all pictures existed. You just have to add 2 pictures and 2 more tests. I'm sure you can manage 🤚